web/assets/dropdown-menu.js

```diff
@@ -1,4 +1,22 @@
+import MOJFrontend from "@ministryofjustice/frontend/moj/all.js";
+
 export default function dropdownMenu() {
+  new MOJFrontend.ButtonMenu({
+    container: $('[data-module="app-button-menu"]'),
+    mq: "(min-width: 200em)",
+    buttonText: "Menu",
+    buttonClasses:
+      "govuk-button--secondary moj-button-menu__toggle-button--secondary",
+  });
```

Comment:
Should check that the container element exists on the page before applying `MOJFrontend.ButtonMenu`, so that it doesn't try to initialise it on other pages (which is what's causing the error in Cypress).
```suggestion
  $container = $('[data-module="app-button-menu"]');
  if ($container.length > 0) {
    new MOJFrontend.ButtonMenu({
      container: ,
      mq: "(min-width: 200em)",
      buttonText: "Menu",
      buttonClasses:
        "govuk-button--secondary moj-button-menu__toggle-button--secondary",
    });
  }
```